### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by Hirushan-N**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Description
+<!-- A clear and concise description of the feature. -->
```
Comment (dileepadev-old) [inline .github/pull_request_template.md:2]: Add details

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Related Issue(s)
+<!-- List any related issues that the feature addresses or solves. -->
```
Comment (dileepadev-old) [inline .github/pull_request_template.md:6]: Add details

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Type of change
+Please delete options that don't apply.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Changes Made
+<!-- Describe the changes made to implement the feature. -->
```
Comment (dileepadev-old) [inline .github/pull_request_template.md:19]: Add details

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Screenshots (optional)
+<!-- Include any relevant screenshots or GIFs to demonstrate the feature. -->
```
Comment (dileepadev-old) [inline .github/pull_request_template.md:23]: Add details

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] All tests pass locally.
+- [ ] Code follows the project's coding conventions.
+- [ ] Changes have been properly documented.
+- [ ] Commit messages are descriptive and follow the project's commit message conventions.
+- [ ] Changes have been reviewed by at least one other developer.
+- [ ] Relevant documentation has been updated or created.
+- [ ] Any necessary migration steps have been documented.
+
+## Anything Else?
+<!-- Is there anything else that you would like to mention about these changes? For example, do they require any additional steps or documentation for future maintenance? -->
```
Comment (dileepadev-old) [inline .github/pull_request_template.md:38]: Add details

**Review by dileepadev-old — CHANGES_REQUESTED** [review]
@Hirushan-N Can you please add more details and point out the PR descriptions. 😊

File: .github/pull_request_template.md (hunk 1 above)
Comment (Hirushan-N) [reply .github/pull_request_template.md:2]: Dit it

File: .github/pull_request_template.md (hunk 2 above)
Comment (Hirushan-N) [reply .github/pull_request_template.md:6]: Dit it

File: .github/pull_request_template.md (hunk 3 above)
Comment (Hirushan-N) [reply .github/pull_request_template.md:19]: Dit it

File: .github/pull_request_template.md (hunk 4 above)
Comment (Hirushan-N) [reply .github/pull_request_template.md:23]: Dit it

File: .github/pull_request_template.md (hunk 5 above)
Comment (Hirushan-N) [reply .github/pull_request_template.md:38]: Dit it

**Review by dileepadev-old — APPROVED** [review]
Good contribution. @Hirushan-N 👍🏼🥳

**Pull request merged**